### PR TITLE
Da focus ring

### PR DESCRIPTION
**Pull request by danamansana**
This addresses the funny focus ring issue by removing default focusing and adding in desired focus behavior. As per my (Daniel) discussion with Kang, there might be a better alternative solution

### CODE REVIEW

**Comment by ktp242** [comment]
I think I figured what this solution is about. And I have to revise my observation for this issue.

Context:
The weird focus ring is from `<main>`. And it is because some elements are not visually wrapped by their parent elements.

This PR's solution:
1) We canceled every focus ring. So the focus ring from `<main>` is not visible anymore. Along with all the other focus rings.
2) To create a focus ring specifically for the element we want, in this case `. tabPanel`, we assign a focus ring style to its child `<ol>` (Notice: not `. tabPanel` itself), so when we click `<main>` we will see `<ol>` being focused.

I think this solution has a big potential risk that is the focus ring is misleading what we are really focusing on. It could cause problem while we are trying to manage our tab order in the future. Also the way to cancel all the focus rings is not ideal. When you do skip the content, you get nothing focused because we canceled all focus rings.

If you look closer to the element of `<main>`, you will see it has a `min-height: 860px`. I think it is the one cause the shape. If we can remove the rule safely, we will get a rectangle focus ring. I don't see any side effect so far but we should do more tests. 

Also, there is a element, `.gs-results-divideLineIcon svgIcon`, I don't think we need it anymore. It is an `<svg>`, I don't see it serves any purposes. I think it was a line we used before. We can make the focus ring "prettier" by removing it.

File: src/app/components/Results/Results.jsx [hunk 1]
```diff
@@ -372,6 +372,7 @@ class Results extends React.Component {
           <div
             tabIndex="0"
             role="tabpanel"
+            className="tabPanel"
```
Comment (ktp242) [inline src/app/components/Results/Results.jsx]: The name doesn't really reflect what it is. It also include the result list. I know the role uses the same value but I think that is for machine to read while class is for human.

File: src/client/styles/main.scss [hunk 2]
```diff
@@ -33,6 +33,11 @@ Import style rules from App components
 @import "components/PaginationButton";
 @import "components/_ReturnLink";
 
+*:focus {
+  outline: none;
```
Comment (ktp242) [inline src/client/styles/main.scss]: I don't think we should cancel the focus outline on everything. Which elements's focus rings do you really want to make it invisible?

The weird focus ring comes from `<main>`.

**Comment by danamansana** [comment]
Removing the min-height property does 
1) Get rid of the funny shape of the focus ring
2) Get rid of the focus ring when we click on whitespace to the left or right
However it also
1) Does not get rid of the focus ring when we click on gray space to the right of the nav
2) Most important, we lose the white area when we first land on the page

**Comment by danamansana** [comment]
Also, I agree that removing all focus is dangerous

**Pull request closed without merging**